pyccel/parser/semantic.py

```diff
@@ -6,7 +6,8 @@
 
 # pylint: disable=R0201, missing-function-docstring
 
-from itertools import chain
+from collections import OrderedDict
```

Comment:
I don't see where you use this, but you probably don't need it. `dict` is ordered since python3.6 and python3.5 is now obsolete so we no longer need to use it

Reply:
@MostafaMamouni don't forget about this (very) old comment